bugz/cli.py

```diff
@@ -184,6 +184,14 @@ def prompt_for_bug(settings):
     else:
         log_info('Enter alias (optional): %s' % settings.alias)
 
+    if not hasattr(settings, 'cf_stabilisation_atoms'):
+        package_list_msg = 'Enter the Package list for this bug (optional): '
+        line = input(package_list_msg)
+        if len(line):
+            settings.cf_stabilisation_atoms = line
```

Comment:
My issue here is the fact that the package list can contain multiple lines. Maybe we should `input` in a loop until an empty line?